vagrant/Vagrantfile

```diff
@@ -0,0 +1,139 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+require 'socket'
+
+hostname = Socket.gethostname
+localmachineip = IPSocket.getaddress(Socket.gethostname)
+puts %Q{ This machine has the IP '#{localmachineip} and host name '#{hostname}'}
+
+# Vagrantfile API/syntax version. Don't touch unless you know what you're doing!
+VAGRANTFILE_API_VERSION = '2'
+
+# Validate required plugins
+REQUIRED_PLUGINS = %w(vagrant-registration vagrant-hostmanager landrush)
+errors = []
+
+def message(name)
+  "#{name} plugin is not installed, run `vagrant plugin install #{name}` to install it."
+end
+# Validate and collect error message if plugin is not installed
+REQUIRED_PLUGINS.each { |plugin| errors << message(plugin) unless Vagrant.has_plugin?(plugin) }
+unless errors.empty?
+  msg = errors.size > 1 ? "Errors: \n* #{errors.join("\n* ")}" : "Error: #{errors.first}"
+  fail Vagrant::Errors::VagrantError.new, msg
+end
+
+if ENV['DEPLOYMENT_TYPE'] == 'enterprise'
+  box_name = 'rhel/7.2'
+else
+  box_name = 'centos/7'
+end
```

Comment:
Should probably have an option for Fedora here as well, maybe a combination of DEPLOYMENT_TYPE and HOST_OS, where 'enterprise' is only valid for RHEL and origin could be any of the above.